corehq/apps/cloudcare/static/cloudcare/js/utils.js

```diff
@@ -124,20 +124,62 @@ hqDefine('cloudcare/js/utils', [
         return $container;
     };
 
+    var shouldShowHideLoading = function () {
+        FormplayerFrontend = hqImport("cloudcare/js/formplayer/app");
```

Comment:
Is there any worry of circular dependencies, since these utils are being imported in `formplayer/app.js` as well? (I assume it's fine because there's already another function in here that does the same)

Reply:
this is also fixed now https://github.com/dimagi/commcare-hq/pull/34158/commits/18f6484c08c1be93e8f7ae0dd7772b9cd2d8ca5c